### PR TITLE
Setup store

### PR DESCRIPTION
**Pull request by zunairkhan811**
The Task is implemented according to the following requirement:
Add Redux Toolkit (npm install react-redux @reduxjs/toolkit).
Create a directory that will contain all your Redux logic (/src/redux)
Configure a Redux store (/src/redux/store.js)
Define a slice of state for books that:
Includes an array of books (initial state: empty array)
Includes a reducer that adds a book
Includes a reducer that removes a book
Define a slice of state for categories that:
Includes an array of categories (initial state: empty array)
Includes a reducer that checks the status and always returns "Under construction" (the initial state should check to that string)

### CODE REVIEW

File: src/redux/books/myBookSlice.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  initialState,
+  reducers: {
+    addBooks: (state, action) => {
+      state.myBooks.push(action.payload);
+    },
+
+  },
+  removedBooks: (state, action) => {
+    state.books = state.books.filter((book) => book.id !== action.payloaded);
+  },
```
Comment (iambenkis) [inline src/redux/books/myBookSlice.js:15]: Good job setting the Redux features , however the add and remove book action do not have any effect on the view of the app , Kindly take a look at the loom video below and fix the issue 👍 

https://www.loom.com/share/37384e1222574807a75c439cf02608b8

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @zunairkhan811  
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/myBookSlice.js (hunk 1 above)
Comment (zunairkhan811) [reply src/redux/books/myBookSlice.js:15]: image

Please Kindly note the instruction I don't have to make changes to the components for this activity

**Review by ShadyShawkat — APPROVED** [review]
## Hi @zunairkhan811, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**